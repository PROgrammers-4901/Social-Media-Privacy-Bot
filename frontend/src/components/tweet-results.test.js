import React from 'react';
import renderer from 'react-test-renderer';

import TweetResults from './tweet-results.component';

it('renders correctly given no input', () => {
    const component = renderer.create(<TweetResults />).toJSON();
    expect(component).toMatchSnapshot();
});

it('renders correctly given results', () => {
    const output = {
        tweet: "1383923017923760131",
        label: "1",
        confidence: "1"
    };

    const component = renderer.create(<TweetResults results = {output} />).toJSON();
    expect(component).toMatchSnapshot();
});