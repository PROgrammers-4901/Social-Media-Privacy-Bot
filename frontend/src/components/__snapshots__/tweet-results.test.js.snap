// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly given no input 1`] = `
<div>
  <figure
    className="text-center"
  >
    <blockquote
      className="blockquote"
    />
  </figure>
</div>
`;

exports[`renders correctly given results 1`] = `
<div>
  <figure
    className="text-center"
  >
    <blockquote
      className="blockquote"
    />
  </figure>
</div>
`;
